types/ember__component/index.d.ts

```diff
@@ -8,10 +8,11 @@
 
 /// <reference types="jquery" />
 
-import CoreView from "@ember/component/-private/core-view";
-import ClassNamesSupport from "@ember/component/-private/class-names-support";
-import ViewMixin from "@ember/component/-private/view-mixin";
+import CoreView from '@ember/component/-private/core-view';
```

Comment:
Looks like Prettier got itself involved in this file when I saved without me noticing 🙃

Reply:
THE WORST.

(We should do another Prettier pass on all of these at some point. Or: not, and get them into Ember itself. :joy:)